opencti-platform/opencti-front/src/utils/hooks/useGranted.ts

```diff
@@ -58,4 +58,9 @@ const useGranted = (capabilities: string[], matchAll = false): boolean => {
   return numberOfAvailableCapabilities > 0;
 };
 
+export const useIsAdmin = () => {
+  // only admins have access to the platform settings, so it's a good shortcut for now
+  return useGranted([SETTINGS]);
```

Comment:
SETTINGS is not really an admin capability, it only gives access to the settings page, the name of the function "useIsAdmin" might give the wrong idea. Maybe it could be `useGrantedSettings` if we want a shortcut ?

Reply:
Actually this is what we check to protect Admin-only features in the frontend, in various places, so I'm just adding an abstraction layer around it to make it clearer.

What should be the real way to check if a user is admin in your opinion ?

Reply:
Maybe BYPASS ?

Reply:
Well... it's out of scope for this PR I'll just revert it.